addons/vanguard/CfgVehicles_Vehicles.hpp

```diff
@@ -104,7 +104,12 @@ class TACU_Vanguard_V_AFV4_Gorgon: I_APC_Wheeled_03_cannon_F {
     displayName = "AFV-4 Gorgon (Sand)";
     crew = "TACU_Vanguard_U_Contractor_Sand_Driver";
     typicalCargo[] = {"TACU_Vanguard_U_Contractor_Green_Engineer"};
-    hiddenSelectionsTextures[] = {QPATHTOF(data\Vanguard_V_AFV4_Gorgon_CO.paa), "A3\armor_f_gamma\APC_Wheeled_03\data\apc_wheeled_03_ext2_co.paa", "A3\armor_f_gamma\APC_Wheeled_03\data\rcws30_co.paa", "A3\armor_f_gamma\APC_Wheeled_03\data\apc_wheeled_03_ext_alpha_co.paa"};
+    hiddenSelectionsTextures[] = {
```

Comment:
No need for a hiddenSelectionsTextures if it's just a vanilla gorgon at this point (unless you're mixing different vanilla textures)

Reply:
It's using the NATO texture not the AAF one

Reply:
Ah. Isn't there a NATO variant already though that you could use as the base class? Or am I misremembering.
Otherwise all good I think.

Reply:
There is, think it's added by missing units. But this is just future proofing incase we ever remove that (the textures are included in Vanilla anyway)